### DISCUSSION TITLE
Will Apollo Server v3 receive updates now that v4 is released?

### DISCUSSION THREAD

**Discussion by blakek** [apollographql/apollo-server#7091] [Q&A] [answered]
We were just curious:

- will v3 receive updates going forward?
- if so, it is safe to assume the updates will only be security-related?
- do you have a date when v3 will be officially end-of-life?

Thanks y'all 🙏🏻

**Comment by glasser** [comment]
Great questions!

When we released v3 last year, we stopped providing updates to v2 other than security updates and fixes to major regressions caused by changes to other software.  However, we didn't formally write this down, nor did we formally provide an "end of life" date for v2.

We are working on doing a better job of communicating about this going forwards, and are finalizing an official policy answering exactly these questions about v3 and v2. Stay tuned!

**Reply by blakek** [reply]
We noticed v3 now has an EOL date. Is it safe assume v3 will no longer receive any updates (including security-related)? Thanks!

**Reply by glasser** [reply, marked as answer]
Does this page answer your questions? https://www.apollographql.com/docs/apollo-server/previous-versions
